<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLCDNumber" name="lcdNumberFrequency">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>291</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberDutyCycle">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>221</width>
      <height>91</height>
     </rect>
    </property>
    <property name="value" stdset="0">
     <double>50.000000000000000</double>
    </property>
   </widget>
   <widget class="QDial" name="dialDutyCycle">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>140</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGetVersion">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Version?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonUpgradeFirmware">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>190</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Upgrade FW</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>240</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Duty Cycle</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>301</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Frequency</string>
    </property>
    <widget class="QDial" name="dialFine">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>111</width>
       <height>101</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>110</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Coarse</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>110</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Fine</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QDial" name="dialCoarse">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>111</width>
       <height>101</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
    <widget class="QSlider" name="verticalSliderRange">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>20</width>
       <height>71</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>4</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonTestTransfer">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>220</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Test Transfer</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>563</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>dialDutyCycle</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberDutyCycle</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>489</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>481</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
